.hlint.yaml

```diff
@@ -23,15 +23,17 @@
 - ignore: {name: "Use <$>"}
 - ignore: {name: "Use uncurry"}
 - ignore: {name: "Redundant $!"}
+- ignore: {name: "Redundant bracket"}
```

Comment:
We should not add this, it will suppress all redundant brackets. Do we want that?